src/components/LoginModal.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
Hey @cupidcoder, consider making the `LoginModal` reusable both in name and implementation. Some other features will be needing it

Reply:
@Chiazokam currently working on this.

Reply:
@Chiazokam This has been worked